packages/playground/src/components/profile-settings-tokens-list.tsx

```diff
@@ -45,8 +46,12 @@ export const ProfileTokensList: React.FC<PageProps> = ({ tokens }) => {
               <TableCell>{token.expires_at ? new Date(token.expires_at).toLocaleString() : 'No Expiration'}</TableCell>
               <TableCell>{timeAgo(new Date(token.issued_at!).getTime())}</TableCell>
               <TableCell className="content-center">
-                <div className="flex gap-1.5 items-center justify-end">
-                  <Icon name="vertical-ellipsis" size={14} className="text-tertiary-background" />
+                <div
+                  className="flex gap-1.5 items-center justify-end cursor-pointer"
+                  onClick={() => {
+                    openAlertDeleteDialog({ identifier: token.identifier!, type: 'token' })
```

Comment:
say if api fails, will we have identifier ??

Reply:
The identifier isn't being fetched dynamically; it's already available when the user attempts to delete. If the identifier is missing, it indicates that the `useTokenListQuery`, which fetches the tokens along with the `identifier`, has failed, and would consequently throw an error there. This identifier has to be `non-null` over here.